File: udpBroadcaster/src/package/fileManagement.py
class FileManagement:

    # read contents of file and return a list of results
    def readFromFile(self, path):
        arr = []
        line = ""
        try:
            with open(path, 'r') as f:
                line = f.read()
            f.close()
        except:
            print("Error in locating file")
        arr = line.split('\n')
        return arr

    def clearFileContent(self, path):
        with open(path, 'w') as f:
            pass
        f.close()
        return

    # writes content by line
    def writeToFile(self, method, path, content):
        with open(path, method) as f:
            f.write(content)
        f.close()

File: classification-server/src/main/java/com/example/demo/service/FileManagement.java
package com.example.demo.service;

import java.util.List;

public interface FileManagement {
	public List<String> readFromFile(String path);
	public boolean clearFileContents(String path);
	public boolean writeToFile(String path, List<String> content);
	public boolean createNewFile(String path);
}
